dali/pipeline/pipeline.cc

```diff
@@ -516,6 +516,11 @@ void Pipeline::PrepareOpSpec(OpSpec *spec) {
     .AddArg("num_threads", num_threads_)
     .AddArg("device_id", device_id_)
     .AddArgIfNotExisting("seed", seed_[current_seed_]);
+  string dev = spec->GetArgument<string>("device");
+  if (dev == "cpu" || dev == "mixed")
+    spec->AddArg("cpu_prefetch_queue_depth", prefetch_queue_depth_.cpu_size);
```

Comment:
I recall, that this snippet was also added in other PR. Please remember to rebase before merging

Reply:
The merge conflict should appear while rebasing it, allowing me to handle it... if git does properly its job :+1: